foo.py

```diff
@@ -3,26 +3,27 @@
 from foo import bar
 from foo import bat
 
-import io  # noqa
 import re  # noqa
 
 
-def foo_one(x, y):
+def foo_one(x, y, z):
     """do foo one"""
     bar.do_x(x)
     bar.do_y(y)
+    bat.do_z(z)
 
 
 def foo_two(z, x, y):
     """do foo two"""
     bat.do_z(z * 12)
     bar.do_x(x).y(y).z(z)
-    bar.do_y(y)
+    bar.do_y(y + 9)
 
 
 def foo_three(a, g, j):
     """do foo three"""
     bat.do_a(a)
```

Comment:
**mike bayer** (zzzeek) wrote:

and didn't do a thing here that github can't see

Reply:
nope, I can see it, still in the diff

Reply:
**mike bayer** (zzzeek) wrote:

touche